supplemental_content/views.py

```diff
@@ -42,3 +45,38 @@ def get_queryset(self):
                 )
             ).distinct()
         return query
+
+
+class SupplementalContentSectionsView(generics.CreateAPIView):
+    def create(self, request, *args, **kwargs):
+        for section in request.data["sections"]:
+            try:
+                new_section, created = Section.objects.get_or_create(
+                            title=section["title"],
+                            part=section["part"],
+                            section_id=section["section"]
+                        )
+            except Exception:
```

Comment:
can we also add a bit of info about what specific thing errored out?  We are going to be troubleshooting this, we should make our lives easier.